op-monitorism/faultproof_withdrawals/runbooks/RUNBOOK.md

```
@@ -0,0 +1,172 @@
+---
+## Alerts
+An incident will be declared upon receiving an alert. The metrics described below trigger various alerts with differing severities. Each alert necessitates specific actions.
+
+### faultproof-withdrawal-forgery-detected
+
+| **Network** | **Severity Level** | **Impact**                      | **Reaction**                                             | **Actions**                                                                                                                                       |
+|-------------|--------------------|---------------------------------|----------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| Mainnet     | SEV1               | Fund loss                       | Immediate action required                                | - Page Faultproof (FP) team<br>- Follow critical incident procedures **NOTE:private procedure**                                                                              |
+| Sepolia     | SEV3               | Fund loss                       | Assess acceptability of loss on Sepolia<br>Investigate the specific withdrawal | - Page Faultproof (FP) team<br>- Follow critical incident procedures  **NOTE:private procedure** |
+
+##### Alert Description
```

Markdownlint

Expected: h4; Actual: h5
Heading levels should only increment by one level at a time

(MD001, heading-increment)